airflow/models/baseoperator.py

```diff
@@ -359,6 +363,8 @@ def __init__(
         self.email = email
         self.email_on_retry = email_on_retry
         self.email_on_failure = email_on_failure
+        self._task_tags: Optional[List[str]] = []
+        self.task_tags = task_tags or []
```

Comment:
```suggestion
        self.task_tags = task_tags
```

The setter already handles None.

Reply:
Fixed in e67fd78